backends/php-laravel/app/Support/Http/Requests/AppRequest.php

```
@@ -70,10 +70,11 @@ public function all($keys = null)
     #[\Override]
     public function input($key = null, $default = null)
     {
+        $extended = array_merge(parent::input(), $this->getExtendValue());
```

PHPStan (2.1.54)

[error] App\Support\Http\Requests\AppRequest::input() calls parent::input() but App\Support\Http\Requests\AppRequest does not extend any class.


(class.noParent)